packages/components/anchor/src/anchor.vue

```diff
@@ -134,13 +134,12 @@ const getCurrentHref = () => {
     })
   }
   anchorTopList.sort((prev, next) => prev.top - next.top)
-
   for (let i = 0; i < anchorTopList.length; i++) {
     const item = anchorTopList[i]
     const next = anchorTopList[i + 1]
 
     if (i === 0 && scrollTop === 0) {
-      return ''
+      return props.scrollTop ? item.href : ''
     }
```

Comment:
I found that the problem was caused by this code, and it looked like redundant code. WDYT @Fuphoenixes

Reply:
What do you mean by redundant code?

Reply:
I just do not understand why it is designed this way?